main.md

```diff
@@ -101,18 +101,16 @@ This specification uses the terms "access token", "refresh token",
 
 # Request parameter "authorization_details" {#authz_details}
 
-The request parameter `authorization_details` contains, in JSON notation, an array of objects. Each JSON object contains the data to specify the authorization requirements for a certain type of resource. The type of resource or access requirement is determined by the `type` field, which is defined as follow:
+The request parameter `authorization_details` contains, in JSON notation, an array of objects. Each JSON object contains the data to specify the authorization requirements for a certain type of resource. The type of resource or access requirement is determined by the `type` field, which is defined as follows:
 
 `type`:
-:   The type of authorization details as a string. The value of the `type` field determines the allowable contents of the object which contains it. This field is REQUIRED.
+:   The authorization details type as a string. The value of the `type` field determines the allowable contents of the object which contains it. This field is REQUIRED. Note that the type value is a unique-key in the context of an AS.
```

Comment:
We don't define "unique-key" here or what that really means elsewhere and I think this could be ambiguous. IE: Does it mean you can send only one instance of one "type" in a request? (No)

```suggestion
:   An identifier for the authorization details type as a string. The value of the `type` field determines the allowable contents of the object which contains it and is unique for the described API in the context of the AS. This field is REQUIRED.
```

Reply:
was trying to address this from the review: 

> Minor issues:
> 
> Section 2: What does "The AS MUST ensure that there is no collision 
> between different authorization details types that it supports." mean? 
> How is this a 2119 requirement? I _think_ you're trying to point out 
> that the authorization_details string is a unique-key at any AS and that 
> that has consequences on the API _designer_, but I don't know what is 
> expected of an AS coder here. Some clearer words are needed.

Reply:
and deleted "The AS MUST ensure... " line below.

Reply:
Also went with your  "An identifier for the" suggesting, which has outdated this.

Reply:
Fuck, I misread your suggestion. Sorry, I'm tired (and tired or RAR). It's a good suggestion. I'll incorporate.

Reply:
[7686b00](https://github.com/oauthstuff/draft-oauth-rar/pull/94/commits/7686b00d164d1753b6ab279ecec09b28617dda10) does it.